src/App.css

```diff
@@ -16,15 +16,28 @@ body {
 }
 
 .View--Login {
+  height: 100vh;
   background-image: linear-gradient(to bottom, #fff 400px, transparent 100%),
-    url(https://portal.kidsfirstdrc.org/static/media/background-science.68317e4e.jpg);
+    url(./assets/background-science.jpg);
+    .center.column{
+      min-width: 300px !important;
+    }
 }
 
-.Card--Login {
-  @extend .Card;
-  width: 620px;
+.ui.button.button--login{
+    margin-bottom: 10px;
+    .auth0-logo{
```

Comment:
When the auth0 image changes to the shield icon, this shouldn't be needed.

Reply:
👎 not using the shield icon in the design